druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/JDBCExtractionNamespace.java

```diff
@@ -123,6 +122,11 @@ public boolean isCacheEnabled() {
         return cacheEnabled;
     }
 
+    @Override
+    public ExtractionNameSpaceSchemaType getType() {
+        return ExtractionNameSpaceSchemaType.Protobuf;
```

Comment:
technically JDBCExtractionNamespace is not using any schema. it is storing String as it is. you may want to have `None` as additional type for this case.

Reply:
done